src/PollinationSDK/Helper/AuthHelper.cs

```diff
@@ -79,21 +120,22 @@ public static async Task SignInWithApiAuthAsync(string apiAuth, Action ActionWhe
         }
 
 
-        private static async Task<string> PollinationSignInAsync(bool devEnv = false)
+        public static async Task<AuthResult> PollinationSignInAsync(bool devEnv = false)
```

Comment:
Do you need to make this public?

Reply:
Oh, good catch. That was just for my test app. Will update.

Reply:
Fixed in cd1c8a2